azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/runtime/WebAppRuntimeHandler.java

```diff
@@ -15,14 +15,13 @@
 import com.microsoft.azure.management.appservice.WebApp;
 import com.microsoft.azure.management.appservice.WebContainer;
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
-import com.microsoft.azure.maven.webapp.handlers.RuntimeHandler;
 import com.microsoft.azure.maven.webapp.utils.WebAppUtils;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.logging.Log;
 import org.apache.maven.settings.Settings;
 import org.codehaus.plexus.util.StringUtils;
 
-public abstract class BaseRuntimeHandler implements RuntimeHandler {
+public abstract class WebAppRuntimeHandler extends com.microsoft.azure.maven.handlers.runtime.BaseRuntimeHandler<WebApp> {
```

Comment:
why not import com.microsoft.azure.maven.handlers.runtime.BaseRuntimeHandler to avoid long line of code?

Reply:
My mistake, this class used to called BaseRuntimeHandler and i didn't change the extends class after refactor it, will fix it in later commit.